.github/workflows/cd.yml

```diff
@@ -0,0 +1,38 @@
+name: NAME-EXAMINATION UI CD
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "web/**"
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment"
+        required: true
+        type: choice
+        options:
+        - dev
+        - test
+        - sandbox
+        - prod
+      tagname:
+        description: "Specify a previous version (git tag) to deploy"
+        required: false
+        default: ""
+
+jobs:
+  analytics-ui-cd:
+    permissions:
+      contents: 'write'
+      id-token: 'write'  
+    uses: bcgov/name-examination/.github/workflows/cd.yaml@main
+    with:
+      environment: ${{ inputs.environment }}
+      tagname: ${{ inputs.tagname }}
+      working_directory: "./web"
```

Comment:
Don't need to set working_directory for this project. It will run everything from root.